code-studio/src/js/components/playzone.jsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * @overview React for the End-of-Stage Experience
+ */
+/* global React */
+var color = require('@cdo/apps/color');
+
+export class PlayZone extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.links = [{
+      href: "/projects/artist/new",
+      img: "/shared/images/courses/logo_tall_artist.jpg",
+      title: this.props.i18n.t('components.playzone.artist_title'),
+      className: "artist",
+      description: this.props.i18n.t('components.playzone.artist_description'),
+    }, {
+      href: "/projects/playlab/new",
+      img: "/shared/images/courses/logo_tall_playlab.jpg",
+      title: this.props.i18n.t('components.playzone.playlab_title'),
+      className: "playlab",
+      description: this.props.i18n.t('components.playzone.playlab_description'),
+    }, {
+      href: "/projects",
+      img: "/shared/images/courses/logo_tall_applab.jpg",
+      title: this.props.i18n.t('components.playzone.projects_title'),
+      className: "projects",
+      description: this.props.i18n.t('components.playzone.projects_description'),
+    }];
+
+    this.styles = {
+      container: {margin: '20px'},
+      primaryHeader: {fontSize: '200%'},
+      secondaryHeader: {color: color.charcoal},
+      courseblockContainer: {width: '720px', paddingTop: '20px'},
+      continueButton: {marginTop: '20px', marginRight: 0}
+    };
+  }
+
+  render() {
+    return (
+      <div style={this.styles.container}>
+        <h1 style={this.styles.primaryHeader}>{this.props.i18n.t('components.playzone.primary_header', {stageName: this.props.stageName})}</h1>
+        <h4 style={this.styles.secondaryHeader}>{this.props.i18n.t('components.playzone.secondary_header')}</h4>
+        <div className="center" style={this.styles.courseblockContainer}>{this.links.map(link =>
+          <div key={link.className} className="courseblock-noaction courseblock-span3 courseblock-tall">
+            <a href={link.href}>
+              <div className="imgspan">
+                <img height="120px" src={link.img} alt={`Logo tall ${link.className}`} />
+              </div>
+              <div className="heading">
+                <h3>{link.title}</h3>
+              </div>
+              <div className="smalltext">{link.description}</div>
+            </a>
+          </div>
+        )}</div>
+        <div className="farSide">
+          <button id="ok-button" onClick={this.props.onContinue} style={this.styles.continueButton}>
```

Comment:
Our style guide says "Components with many attributes should have one per line". This isn't something that we're super strict about I don't think, but you might find some of our JSX gets easier to read (albeit longer) if you take that approach.